src/debugging/Inspector/Inspector/NativeScript Inspector/ViewController.m

```diff
@@ -2,26 +2,31 @@
 @import AppKit;
 @import WebKit;
 @import JavaScriptCore;
+#import <WebKit/WebPreferencesPrivate.h>
 #import "InspectorFrontendHost.h"
 
-@implementation ViewController
+@implementation ViewController {
+    NSString* _mainFileName;
```

Comment:
IMO if we have this as fields, we don't need to have parameterized methods ( ```update``` could just use ```self->_projectName``` ) but it's not that important